mozdownload/scraper.py

```diff
@@ -288,7 +288,7 @@ def _download():
                                           maxval=max_value).start()
 
                 with open(tmp_file, 'wb') as f:
-                    for chunk in iter(lambda: r.raw.read(CHUNK_SIZE), ''):
+                    for chunk in r.iter_content(CHUNK_SIZE):
```

Comment:
With `unicode_literals` I got a unicode conversion failure when trying to download the 42.0 release. As given by the requests documentation you should better use its own iterator to prevent the usage of raw strings.